pydundas/rest/constant.py

```diff
@@ -0,0 +1,169 @@
+        "PRIVILEGE_UNSAFE_TRANSFORM_ACCESS": "b308317a-35f2-4c6b-a19a-9dbf59a7ef2a",
+        "PROJECTS_ROOT_FOLDER_ID": "b1668d22-840e-4fa1-80df-958d58bff12d",
+        "RAGGED_ALL_MEMBERS_TOKEN_ID": "6b48acc0-54a7-4d8d-aea9-0e1c912918ad",
+        "RAGGED_LEAF_MEMBERS_TOKEN_ID": "fd0e6434-7ca7-4ad3-b64d-fcd8c3cdfb83",
+        "REBUILD_INDEXS_JOB_ID": "4323f2f1-a8b8-459b-9ef4-52f3a7914300",
+        "RECYCLE_BIN_ROOT_FOLDER_ID": "0379ac09-8e16-4f27-b4e6-a7e459180864",
+        "SERVER_ONLY_USER_DATA_KEY_PREFIX": "227c3c21-a15f-44e1-ab01-2a3b1480f25f",
+        "STANDARD_CREATEFILE_DELIVERY_PROVIDER_ID": "3b192991-f37f-4311-940b-f234e62ba471",
+        "STANDARD_CSV_EXPORT_PROVIDER_ID": "07d7e096-ff2b-4da2-84a5-58b27fb67cdc",
+        "STANDARD_DATA_INPUT_TRANSFORM_ID": "a7de0d91-177b-407e-976d-5092ba68ed47",
+        "STANDARD_EMAIL_DELIVERY_PROVIDER_ID": "9effc739-a6be-4272-a63f-375e1645d6a7",
+        "STANDARD_EXCEL_EXPORT_PROVIDER_ID": "679e6337-48aa-4aa3-ad3d-db30ce943dc9",
+        "STANDARD_EXPORT_NAME_PREFIX_ID": "ob95611baftw5fybc33pkoohkh",
+        "STANDARD_IMAGE_EXPORT_PROVIDER_ID": "9a15b21e-fa9e-4ece-8b31-9e12f1c6134a",
+        "STANDARD_PDF_EXPORT_PROVIDER_ID": "5752eb39-40b5-4d79-b96b-9f9297c67193",
+        "STANDARD_POWERPOINT_EXPORT_PROVIDER_ID": "b58d4c94-9f4f-4d38-800b-2b5d4d348625",
+        "STANDARD_PYTHON_GENERATOR_TRANSFORM_ID": "76588681-98D0-453A-9C25-7A56AD42C222",
+        "STANDARD_RESULT_TRANSFORM_ID": "b54baf74-43e7-4941-b18a-3ab5b2591480",
+        "STANDARD_R_GENERATOR_TRANSFORM_ID": "842A090C-0CB6-41ED-8F6C-0E4D6A9EDB64",
+        "STANDARD_USERS_GROUP": "00d544b9-bfad-434a-acd9-f2bd7a93f6f8",
+        "SYSTEM_ADMINISTRATORS_GROUP": "23422323-0CD3-42B7-BAA6-8A73E00BA6C1",
+        "SYSTEM_CLEANUP_JOB_ID": "3c8bce75-45cd-4839-a406-a0068eb47f20",
+        "SYSTEM_SERVICE_ID": "90ac7b78-62e7-41d9-b16b-c298aadfade5",
+        "TEMPLATE_LEFT_SPLIT_ID": "d4d8d79d-d274-4f73-ad76-a0271e378d39",
+        "TEMPLATE_QUADRANT_ID": "524ce722-c6c8-4dbf-a872-5f5f59738a40",
+        "TEMPLATE_RIGHT_SPLIT_ID": "aa25fc89-9606-4b5f-bfe3-d416b7c704cc",
+        "TEMPLATE_THREE_COLUMN_ID": "3b331d52-c753-4485-adb6-45f5d4b2d520",
+        "TEMPLATE_THREE_ROW_ID": "df7b59ff-5644-49c3-906c-b0b2174e4b2b",
+        "TEMPLATE_TWO_COLUMN_ID": "4b8644e0-0a09-4bd6-969e-4c552b01362d",
+        "TEMP_FOLDER_ID": "76fb5175-b732-4251-a96e-b8368887fa4e",
+        "TENANT_PROJECTS_ROOT_FOLDER_ID": "710c07a1-7edd-4609-ad2f-8e1426ea0a46",
+        "TREEMAP_SIMPLIFICATION_RULE_ID": "6affc832-e616-b42f-fee3-c835d6d65361",
+        "USER_PROJECTS_ROOT_FOLDER_ID": "587ebce0-be40-415b-aefc-ab520fc38ba6",
+        "USER_SETTINGS_ADMIN_SECTION": "adminSectionSettings",
+        "VIEW_PERSONALIZATION_SHORT_LINK_CATEGORY_ID": "25f614d2-eaa1-4888-bf25-fb0461cc8aa9",
+        "WEB_APP_MODULE_ID": "21259d43-ee4c-4b06-b73f-c9c2cdce60c6",
+    }
+
+    def __init__(self, *irrelevantargs):
+        # Move constants one level up for easy access.
```

Comment:
What you're basically saying is that this should be a generated class.
Continuing on that train of thought:

You've gone 'meta' because you didn't like the fact that you had to copy/paste boiler plate code. Wouldn't code generation be able to solve that same problem without the need for the added complexity?

Reply:
No easy way to generate with pure Rest api.